docs-src/concepts/what-is-a-temporal-sdk.md

```diff
@@ -50,7 +50,6 @@ The following SDKs are in alpha/pre-alpha development stages, but are not yet su
 
```

Comment:
Because the .NET SDK is now in beta, I suggest changing the introductory sentence to something like the following:

`The following SDKs are in development and do not yet have developer's guides:`

Reply:
+1 we should adjust this too -

Reply:
Issue moved to [EDU-1068](https://temporalio.atlassian.net/browse/EDU-1068).